editor/js/Sidebar.Scene.js

```diff
@@ -143,8 +143,9 @@ Sidebar.Scene = function ( editor ) {
 	var fogType = new UI.Select().setOptions( {
 
 		'None': 'None',
-		'Fog': 'Linear',
-		'FogExp2': 'Exponential'
+		'Fog': 'Ranged',
```

Comment:
This time I renamed "Linear" to "Ranged". Maybe that is better. But it is inconsistent.

Reply:
It is no longer inconsistent when using the description "smooth ranged" in the docs.